README.md

```diff
@@ -1,7 +1,9 @@
 # TPEN28
 ![](http://t-pen.org/TPEN/images/tpen_logo_header.jpg)
 
-The Transcription for Paleographical and Editorial Notation (T‑PEN) project is coordinated by the Center for Digital Theology at Saint Louis University (SLU) and funded by the Andrew W. Mellon Foundation and the NEH. The Electronic Norman Anonymous Project developed several abilities at the core of this project's functionality.
+The Transcription for Paleographical and Editorial Notation (T‑PEN) project was coordinated by the Center for Digital Theology at Saint Louis University (SLU) and funded by the Andrew W. Mellon Foundation and the NEH. The Electronic Norman Anonymous Project developed several abilities at the core of this project's functionality.
+
+T-PEN is currently (2022) maintained by the Research Computing Group at Saint Louis University.  T-PEN is no longer funded.
```

Comment:
It is funded, but not by a monolithic program. I think we can drop the date and just say it is maintained by us.